templates/create-update.html

```diff
@@ -261,15 +261,78 @@ <h2>Add optional information</h2>
                      placeholder="If applicable"
                      value="{% if asset and asset.google_drive_link %}{{ asset.google_drive_link }}{% endif %}" />
             </div>
-            <div class="p-section--shallow" data-name="salesforceCampaignId">
+
+            <div class="p-section--shallow" data-name="salesforceCampaigns">
+              {% set existing_campaigns = [] %}
+              {% if asset and asset.salesforce_campaigns | length %}
+                {% for campaign in asset.salesforce_campaigns %}
+                  {% set _ = existing_campaigns.append({
+                    "id": campaign.id,
+                    "name": campaign.name
+                  }) %}
+                {% endfor %}
+              {% elif form_session_data and form_session_data.salesforce_campaigns | length %}
+                {% for campaign in form_session_data.salesforce_campaigns %}
+                  {% set _ = existing_campaigns.append({
+                    "id": campaign.id,
+                    "name": campaign.name
+                  }) %}
+                {% endfor %}
+              {% endif %}
+              <div class="p-search-and-filter js-campaign-search">
+                <label class="p-heading--5 u-no-margin--bottom" id="salesforceCampaigns">Salesforce Campaigns</label>
```

Comment:
```suggestion
                <label class="p-heading--5 u-no-margin--bottom" id="salesforceCampaigns">Salesforce Campaigns</label>
```
This label is not associated with any input

Reply:
It is associated with this input, line 300
```
                  <input autocomplete="off"
                           class="p-search-and-filter__input js-campaign-input"
                           id="search"
                           placeholder="Start typing a campaign name"
                           type="text"
                           value=""
                           aria-labelledby="salesforceCampaigns" />
```

Reply:
My previous comment was incorrect, it has _two_ labels associated with it

Reply:
This and it may cause confusion for screen reader uses